pkg/reconciler/common/unstructured.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2020 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
+
+import (
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+// NamespacedResource is an unstructured resource with the given apiVersion, kind, ns and name.
+func NamespacedResource(apiVersion, kind, ns, name string) *unstructured.Unstructured {
+	resource := &unstructured.Unstructured{}
+	resource.SetAPIVersion(apiVersion)
+	resource.SetKind(kind)
+	resource.SetNamespace(ns)
+	resource.SetName(name)
+	return resource
+}
+
+// ClusterScopedResource is an unstructured resource with the given apiVersion, kind and name.
+func ClusterScopedResource(apiVersion, kind, name string) *unstructured.Unstructured {
```

Comment:
Same as above

Reply:
true, i have made the necessary changes. :+1: